.changeset/quiet-impalas-chew.md

```diff
@@ -0,0 +1,8 @@
+---
+'@aws-amplify/ui-react': patch
```

Comment:
with the new attributes being added and the structure of some of the components being updated do we want this to be a minor bump instead of a patch?

Reply:
Good point...

I checked the [semver doc](https://semver.org/#spec-item-7)

> Minor version Y (x.Y.z | x > 0) MUST be incremented if new, backwards compatible functionality is introduced to the public API. It MUST be incremented if any public API functionality is marked as deprecated. It MAY be incremented if substantial new functionality or improvements are introduced within the private code. It MAY include patch level changes. Patch version MUST be reset to 0 when minor version is incremented.

Since we don't have any new features in this PR, I think it's not a MUST.